src/main/resources/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy/config.groovy

```diff
@@ -0,0 +1,7 @@
+package org.jenkinsci.plugins.durabletask.executors.OnceRetentionStrategy
+
+def f = namespace(lib.FormTagLib);
+
+f.entry(title: "Idle timeout", field: "idleMinutes") {
+    f.number(default: 0)
```

Comment:
(and we use our own because we were using it since 2011... if it ain't broke don't fix it...)